plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardProjectConfig.java

```diff
@@ -78,4 +79,15 @@ public void setAppEngineLibraries(Collection<Library> libraries) {
     this.appEngineLibraries = new ArrayList<>(libraries);
   }
 
+  String getServiceName() {
+    if (serviceName == null) {
+      return "default"; // NON-NLS-1
```

Comment:
the default behavior in gcloud and the app engine docs all just leave 'service' out when targetting 'default'.